nBTC/sources/redeem_request.move

```diff
@@ -75,7 +75,7 @@ public struct SolvedEvent has copy, drop {
 public struct RequestSignatureEvent has copy, drop {
     redeem_id: u64,
     sign_id: ID, // IKA sign session ID
-    input_idx: u32,
+    input_id: u32,
```

Comment:
I think this should be idx (index), not an id

Reply:
ah, because we easy confuse with object ID.